lora-phy/src/lib.rs

```diff
@@ -95,6 +95,10 @@ where
     }
 
     /// Process an IRQ event and return the new state of the radio
+    ///
+    /// # Warning
```

Comment:
Maybe use `# Safety` instead of warning? As per https://std-dev-guide.rust-lang.org/policy/safety-comments.html

Reply:
Actually, it doesn't matter. As both "Safety" and "Warning" look the same in docs.